src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -157,7 +191,16 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
 
                     foreach (var role in rolesToRemove)
                     {
-                        await _userRoleStore.RemoveFromRoleAsync(user, _userManager.NormalizeName(role), default(CancellationToken));
+                        // Make sure we always have at least one administrator account
+                        if (usersOfAdminRole.Count == 1 && usersOfAdminRole.First().UserName == user.UserName && role == "Administrator")
+                        {
+                            _notifier.Warning(H["Cannot remove administrator role from unique administrator."]);
```

Comment:
Still the "unique" is not make sense to me ;) but I don't mind if the its fine for the others